webui/Makefile

```diff
@@ -1,22 +1,27 @@
-.PHONY: default get-deps build clean test fmt check
-
-default: get-deps build
-
-clean:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
-
-fmt:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
-
-check:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
-
+.PHONY: all
+all:
+	make get-deps
+	make build
+
+.PHONY: clean
+clean: clean-react clean-elm clean-tests clean-api-ts-sdk
+.PHONY: clean-%
+clean-%:
+	$(MAKE) -C $* clean
+
+.PHONY: fmt
+fmt: fmt-react fmt-elm fmt-tests
+.PHONY: fmt-%
+fmt-%:
+	$(MAKE) -C $(subst -,/,$*) fmt
```

Comment:
Should probably get rid of the other uses of `subst` as well to keep it consistent.